examples/rlhf/models/reward.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+
+import torch
+import torch.nn as nn
+from tensordict.nn import TensorDictModule
+from transformers import GPT2LMHeadModel, GPT2TokenizerFast
+
+
+class RewardModel(nn.Module):
```

Comment:
missing docstring
Is this reusable or highly tailored to the current project?
We could incorporate it in torchrl and add some tests